docs/operator-manual/application.yaml

```diff
@@ -18,6 +18,7 @@ spec:
     path: guestbook
 
     # helm specific config
+    chart: chart-name  # Set this when pulling directly from a Helm repo. DO NOT set for git-hosted Helm charts.
```

Comment:
:+1: 

I think the `source` part of the application requires even more clarification for Helm. For example, we should document that `repoURL` can be either pointing to a Helm chart repository or to a Git repository. `targetRevision` also has a slightly different semantic between Helm and Git repositories.

Reply:
@jannfis thoughts on the comments I've added?

Reply:
Thanks! LGTM.